tests/api_fastapi/core_api/routes/public/test_assets.py

```diff
@@ -523,6 +523,32 @@ def test_should_respond_404(self, test_client):
         assert response.json()["detail"] == "Queue event with dag_id: `not_exists` was not found"
 
 
+class TestDeleteDagDatasetQueuedEvents(TestQueuedEventEndpoint):
+    @pytest.mark.usefixtures("time_freezer")
+    def test_should_respond_204(self, test_client, session, create_dummy_dag):
+        dag, _ = create_dummy_dag()
+        dag_id = dag.dag_id
+        self.create_assets(session=session, num=1)
+        asset_id = 1
+        self._create_asset_dag_run_queues(dag_id, asset_id, session)
+
+        response = test_client.delete(
+            f"/public/dags/{dag_id}/assets/queuedEvent",
+        )
+
+        assert response.status_code == 204
```

Comment:
I think we should also check the event is deleted

Reply:
Sure, that would be a nice validation to have.

Reply:
Pushed a fix, can you check?